scss/_iconsprite.scss

```diff
@@ -0,0 +1,114 @@
+%svg-common {
```

Comment:
I certainly appreciate the effort to continue using a sprite map but I'm wondering about the maintenance burden. What do you think about loading the individual svg files directly instead? With http/2 any static assets like these often being served up by a proxy that supports it (e.g. CloudFlare), there probably isn't much of a gain in using a sprite map - especially since BetterNavigator isn't seen by the end users.

If you do remove the sprite feature, we might not even need sass. Might be nice to fully modernise the css by using css variables and css nesting, with perhaps [PostCSS](https://github.com/postcss/postcss-cli) or [Lightning CSS](https://lightningcss.dev/transpilation.html) to transpile for older browsers. But not a requirement.